website/views/options/social/index.jade

```diff
@@ -39,7 +39,7 @@ script(type='text/ng-template', id='partials/options.social.party.html')
       p
         =env.t('startLFG')
         |&nbsp;
-        a(href="http://habitrpg.wikia.com/wiki/Board:The_Archery_-_LFG")=env.t('linkLFG')
+        a(href="https://habitrpg.com/#/options/groups/guilds/f2db2a7f-13c5-454d-b3ee-ea1f5089e601")=env.t('linkLFG')
```

Comment:
"To advertise  your new party or find one to join, join the public Archery Guild." ??